ONSdigital/ras-frontstage
# RAS-1298: Add a new survey page

# What and why?
As part of the user journey improvement work, there is a requirement to update and improve the add survey section. This PR achieves this.

# How to test?
- Run tests locally
- Deploy this PR to your local env and run acceptance tests
- check that it matches the requirements laid out in the Jira ticket

# Jira
https://jira.ons.gov.uk/browse/RAS-1298
